lib/main_ltp.pm

```diff
@@ -112,6 +112,20 @@ sub stress_snapshots {
     }
 }
 
+sub prepare_target {
```

Comment:
since you are introducing this new function, would it make sense to define it as `load_tuned_tests`? or it will be used for other things in the future?

Reply:
I would find it confusing in this case. No matter if it will be used or not in other places - function name suppose to say what it actually doing and not was the reason to create it. From this perspective function have good enough name

Reply:
The function definition is side effect of existence `main_ltp.pm`. I'm just reusing function from `main.pm`. I already tried to move it from `main.pm` to `main_common.pm` and basically reuse it in `main_ltp.pm`, but.... same function is defined in opensuse `main.pm` and is little bit different. It would be nice to have it only at once place, but I don't want to mess with it at the moment (probably will create separte poo to sort it out).  

This function is quite helpfull, it will prepare target for test, either from image or it does installation. For example we can reuse it by trinity and run it on spvm (if we want).